### PR TITLE
addRegistrationLayout

### PR DESCRIPTION
**Pull request by ghilbing**
Layout for registration

### CODE REVIEW

File: app/src/main/res/layout/activity_registration.xml [hunk 1]
```diff
@@ -1,9 +1,65 @@
...
+        android:layout_marginTop="22dp" />
+
+    <EditText
+        android:id="@+id/etEmail"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="26dp"
+        android:ems="10"
+        android:hint="@string/email"
+        android:inputType="textPersonName"
```
Comment (00ec454) [inline app/src/main/res/layout/activity_registration.xml]: this should be type email

File: app/src/main/res/layout/activity_registration.xml [hunk 2]
```diff
@@ -1,9 +1,65 @@
...
+        android:layout_below="@+id/etLastName"
+        android:layout_alignLeft="@+id/etLastName"
+        android:layout_alignStart="@+id/etLastName" />
+
+    <Button
+        android:id="@+id/button"
```
Comment (00ec454) [inline app/src/main/res/layout/activity_registration.xml]: can you make this button id something like `@+id/btnRegister`

**Comment by ghilbing** [comment]
Yes, of course


2017-10-12 12:32 GMT-07:00 Dharmesh Gohil <notifications@github.com>:

**Comment by m3libea** [comment]
For me is ready to merge. Only one comment Gretel, It is better if the message of the commits are more descriptive about what are you doing there :)

**Comment by ghilbing** [comment]
Ok, thanks for your feedback, I am learning a lot with both of you!

2017-10-12 15:53 GMT-07:00 Rocío Romero <notifications@github.com>:

**Pull request merged**